src/main/java/org/eclipse/pass/schema/SchemaFetcher.java

```diff
@@ -0,0 +1,55 @@
+package org.eclipse.pass.schema;
```

Comment:
Thanks for the comment. I have reviewed this logic and I think this approach might be better but please let me know:

The repository URIs will be sent as parameters to this service. Repository PASS entities will be created from these URIs.

The schemas relevant to these repositories can be accessed via the Repository's getSchemas() method.
This method will return the schema $id values. However, since they do not need to be fetched over the network, the $id http addresses should not be used for loading resources. The last 2 parts of the schema URIs (between '/'s) can be used to load the local json file as a resource. 

Also should the name of this class be changed? I'm not sure since it is still retrieving schemas, but not fetching them over the network.